src/lib/transition_handler/processor.ml

```diff
@@ -241,7 +241,9 @@ let process_transition ~context:(module Context : CONTEXT) ~trust_system
       (* TODO: only access parent in transition frontier once (already done in call to validate dependencies) #2485 *)
       [%log internal] "Find_parent_breadcrumb" ;
       let parent_breadcrumb =
-        Transition_frontier.find_exn frontier parent_hash
+        Transition_frontier.find frontier parent_hash
```

Comment:
Reworked with reporting an issue in block processing instead of shutting down the node with an exception.

I believe this is the right thing to do.